modules/osdk-cli-reference-new.adoc

```diff
@@ -58,14 +58,26 @@ Starting with Operator SDK v0.12.0, the `--dep-manager` flag and support for
 modules.
 ====
 
-.Example usage for Go project
+.Example usage for Go project:
```

Comment:
Why do these two take a colon?

Reply:
I wasn't sure how to handle that there's multiple codeblocks for this Go one, and then one for the Ansible one. Thought maybe the colon made it seem a little more like a psuedo-procedure. But, I'll remove, it's not adding anything really.